tests/test_pretrial_verified_law_rescue.py

```
@@ -0,0 +1,229 @@
+        unverified_claims=[],
+        source_urls=[_ADILET],
+        notes=[],
+    )
+
+
+def _verified_research_with_procedural_noise() -> LegalResearch:
+    research = _verified_research()
+    research.verified_claims.append(
+        verified_claim_line(
+            "Исковое заявление должно содержать обстоятельства и подтверждающие их доказательства.",
+            "статья 148 ГПК РК",
```

Ruff (0.16.2)

[warning] String contains ambiguous `Р` (CYRILLIC CAPITAL LETTER ER). Did you mean `P` (LATIN CAPITAL LETTER P)?

(RUF001)